ftw/contacts/browser/contact.pt

```diff
@@ -0,0 +1,113 @@
+<html lang="en"
+  metal:use-macro="here/main_template/macros/master"
+  xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+  xmlns:metal="http://xml.zope.org/namespaces/metal"
+  xmlns:tal="http://xml.zope.org/namespaces/tal"
+  i18n:domain="ftw.contacts">
+
+  <body>
+    <div metal:fill-slot="content-core">
+      <metal:main-macro define-macro="content-core">
+        <div id="contact-view">
+          <div class="contactInfos">
+            <table class="contacInfo">
+              <tr tal:condition="python: context.firstname and context.lastname">
+                <th i18n:translate="label_name">Name</th>
+                <td>
+                  <tal:title tal:condition="context/academic_title" tal:replace="context/academic_title">Dr.</tal:title>
+                  <tal:name tal:replace="string:${context/lastname} ${context/firstname}">Boss Hugo</tal:name>
+                </td>
+              </tr>
+              <tr tal:condition="context/function">
+                <th i18n:translate="label_function">Function</th>
+                <td tal:content="context/function">Boss</td>
+              </tr>
+              <tr tal:condition="context/organization">
+                <th i18n:translate="label_organization">Organization</th>
+                <td tal:content="context/organization">XYZ GmbH</td>
+              </tr>
+              <tr tal:condition="context/department">
+                <th i18n:translate="label_department">Department</th>
+                <td tal:content="context/department">CIA</td>
```

Comment:
The literal `CIA` is never displayed, because it is replaced with `tal:content` and is thus not necessary. 
I propose to just remove those as there is no value. Just make it:
```xml
<td tal:content="context/department" />
```

Please also take a look at the output of ``./bin/check-translations`` which usually reports such things..